praw/models/reddit/wikipage.py

```diff
@@ -1,5 +1,5 @@
 """Provide the WikiPage class."""
-from typing import TYPE_CHECKING, Any, Dict, Generator, Optional, Union
+from typing import TYPE_CHECKING, Any, Dict, Generator, Iterator, Optional, Union
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Dict, Generator, Optional, Union
```
Iterator isn't needed since `discussions` returns a `ListingGenerator`.